keras_contrib/applications/resnet.py

```diff
@@ -132,7 +132,7 @@ def _shortcut(input_feature, residual, conv_name_base=None, bn_name_base=None):
 
 def _residual_block(block_function, filters, blocks, stage,
                     transition_strides=None, transition_dilation_rates=None,
-                    dilation_rates=(1, 1), is_first_layer=False, dropout=None,
+                    dilation_rates=None, is_first_layer=False, dropout=None,
```

Comment:
This dilation rate is a 2D dimensional multiplier. It is not a list per block. I don't think this change is correct

Reply:
Yep, its a 2D dimensional multiplier, I was trying it as a list as well and it seems to work which confused me. I have made the change, the change was required because initializing a Deep Resnet was throwing a error with the current code.